format-common/src/main/java/io/cdap/plugin/format/input/PathTrackingInputFormatProvider.java

```diff
@@ -21,21 +21,23 @@
 import io.cdap.cdap.etl.api.validation.FormatContext;
 import io.cdap.cdap.etl.api.validation.ValidatingInputFormat;
 
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import javax.annotation.Nullable;
 
 /**
- * Base class for input format plugins that support tracking which file their record came from.
- *
- * @param <T> type of plugin config
- */
+* Base class for input format plugins that support tracking which file their record came from.
+*
+* @param <T> type of plugin config
+*/
 public abstract class PathTrackingInputFormatProvider<T extends PathTrackingConfig> implements ValidatingInputFormat {
   private static final String NAME_SCHEMA = "schema";
   protected T conf;
 
   protected PathTrackingInputFormatProvider(T conf) {
-    this.conf = conf;
+  this.conf = conf;
```

Comment:
fix indentation. Please make sure your IDE settings are correct, this has been a problem in every round of reviews.